doc/examples/registration/plot_register_affine.py

```diff
@@ -0,0 +1,182 @@
+"""
+=====================================
+Affine Registration
+=====================================
+
+In this example, we use image registration to find an affine transformation
+which can be used to align our target and reference images.
+
+The :func:`skimage.measure.register_affine` function uses a Gaussian pyramid,
+a cost function, and an optimization function to find the affine transformation
+that best converts a reference space to a target space. This transformation
```

Comment:
"converts a reference space to a target space" sounds like linear algebra language, but isn't.  Perhaps simply "that best aligns two images."?

Reply:
@stefanv how about? note that "reference" and "target image" is consistent with the rest of the documentation, and more clear imo.

```suggestion
that best aligns a reference image to a target image. This transformation
```